test/e2e/disruptive_test.go

```diff
@@ -9,17 +9,27 @@ import (
 )
 
 func TestBasicThenDriverRestartThenChangeShare(t *testing.T) {
```

Comment:
As a suggestion for next interaction, `TestBasicThenDriverRestartThenChangeShare` could become a "test table", as we have when generating unit-tests.

Reply:
yeah there has been some back and forth within our team @otaviof over that last year or so on the table approach vs. the explicit test case approach, both with unit and e2e tests, in the various repo's we maintain.

based on that prior history, I am at least of the mind that either approach is fine and we can allow the code writer a little freedom  / flexibility

running tests in parallel technically can come for free when using the test per method approach

there are means of running tests in parallel with the table approach as well, but you have to opt into the helpers etc. that facilitate that